src/test/resources/com/atomist/rug/kind/python3/ChangeImports.ts

```diff
@@ -1,30 +1,26 @@
-import {Project} from '@atomist/rug/model/Core'
+import {Project,File} from '@atomist/rug/model/Core'
 import {ProjectEditor} from '@atomist/rug/operations/ProjectEditor'
 import {PathExpression,TreeNode,TypeProvider} from '@atomist/rug/tree/PathExpression'
 import {PathExpressionEngine} from '@atomist/rug/tree/PathExpression'
 import {Match} from '@atomist/rug/tree/PathExpression'
 import {Parameter} from '@atomist/rug/operations/RugOperation'
 
-class Imports implements ProjectEditor {
+class ChangeImports implements ProjectEditor {
     name: string = "Constructed"
     description: string = "Uses single microgrammar"
 
     edit(project: Project) {
       let eng: PathExpressionEngine = project.context().pathExpressionEngine()
 
-      let i = 0
-
-      // eng.with<File>(project, "//File()[/PythonRawFile()//import_from()//FROM[@value='flask']]", n => {
-      //   console.log(`The file path is ${n.path()}`)
-      //   i++
-      // })
-
-      eng.with<TreeNode>(project, "//File()/PythonRawFile()//import_stmt()", n => {
-        console.log(`The node is ${n.value()}`)
-        i++
+      let count = 0
+      eng.with<TreeNode>(project, "//File()/PythonRawFile()//import_from()//dotted_name[@value='flask']", n => {
```

Comment:
If `import_from` is a type, why does it not adhere to our naming standards and be `ImportFrom`?

Same for `import_stmt`.

I made this comment on #222 but it seems to have gotten erased somehow.